File: _static/codes/chap3/convertTemperatureCtoK.cpp
//convertTemperatureCtoK
//----------------------
//Print temperature in Kelvin from input read temperature in Celcius
#include <iostream>
main()
{
  const float zeroC(273.15);
  float tempC,tempK;
  std::cout << "Enter temperature in Celcius :" << std::endl;
  std::cin >> tempC;
  tempK = tempC + zeroC;
  std::cout << tempC << " °C is " << tempK << " K" << std::endl;
}

File: cours_Info/algo/_static/codes/chap3/convertTemperatureCtoK.py
#convertTemperatureCtoK
#----------------------
#Print temperature in Kelvin from input read temperature in Celcius
# -*-coding:utf8 -*
zeroC = 273.15
tempC = input("Enter temperature in Celcius :")
tempC = float(tempC)
tempK = tempC + zeroC
print(tempC, " °C is ", tempK, " K")